cli/schema/cypress.schema.json

```diff
@@ -103,6 +103,19 @@
       "default": "cypress/integration",
       "description": "Path to folder containing integration test files"
     },
+    "experimentalComponentTesting": {
+      "type": "boolean",
+      "default": false,
+      "description": "Enabled experimental component testing, see https://github.com/cypress-io/cypress/issues/5922"
+    },
+    "componentFolder": {
+      "type": [
```

Comment:
could this be an array? it seems like components are more likely than E2E tests to be in multiple folders.